server/src/test/java/org/candlepin/policy/AutobindRulesTest.java

```diff
@@ -2552,7 +2563,7 @@ public void testShouldSelectPoolWhenRoleMatchesButOtherSysPurposeAttributesAndSo
 
         // Candidate pools:
         Product prodMCT1650 = createSysPurposeProduct(null, "myrole", "random_addon",
-            "random_sla", "random_usage");
+            "random_sla", "random_usage", null);
```

Comment:
Same here: This should be "random_service_type" instead of null, since the test expects all attributes other than role to have some random value

Reply:
Done.